cg/store/filters/status_case_sample_filters.py

```diff
@@ -40,5 +40,5 @@ def apply_case_sample_filter(
 class CaseSampleFilter(Enum):
     """Define CaseSample filter functions."""
 
-    GET_SAMPLES_IN_CASE_BY_INTERNAL_ID: Callable = get_samples_in_case_by_internal_id
-    GET_CASES_WITH_SAMPLE_BY_INTERNAL_ID: Callable = get_cases_with_sample_by_internal_id
+    GET_SAMPLES_IN_CASE_BY_INTERNAL_ID: Callable = filter_samples_in_case_by_internal_id
+    GET_CASES_WITH_SAMPLE_BY_INTERNAL_ID: Callable = filter_cases_with_sample_by_internal_id
```

Comment:
Maybe rename these as well? Most homogeneous would be SAMPLES_IN_CASE_BY_INTERNAL_ID and CASES_WITH_SAMPLE_BY_INTERNAL_ID

Reply:
Fixed